src/app/embed/adapters/merge.adapter.ts

```diff
@@ -28,8 +35,33 @@ export class MergeAdapter {
         if (source === NO_EMIT_YET) { break; }
         data.push(source);
       }
-      return getMergedValues(...data);
-    }).filter(hasMinimumParams);
+      return this.getMergedValues(...data);
+    }).filter(this.hasAnyParams).filter(this.hasMinimumParams);
```

Comment:
Note: this now filters for having ANY params.  I thought it might be useful to block the observable entirely, until there's something to tell the player about.